src/frontend/src/pages/ChangeRequestsPage/ChangeRequestsView.tsx

```diff
@@ -37,43 +36,39 @@ const ChangeRequestsView: React.FC = () => {
   };
 
   return (
-    <>
-      <Box style={{ marginBottom: 15 }}>
-        <PageTitle
-          title={'Change Requests'}
-          previousPages={[]}
-          tabs={
-            <Tabs value={tabIndex} onChange={handleTabChange} variant="standard" aria-label="change-request-tabs">
-              <Tab
-                label="Overview"
-                aria-label="overview"
-                value={0}
-                component={RouterLink}
-                to={`${routes.CHANGE_REQUESTS_OVERVIEW}`}
-              />
-              <Tab
-                label="All Change Requests"
-                aria-label="all-change-requests"
-                value={1}
-                component={RouterLink}
-                to={`${routes.ALL_CHANGE_REQUESTS}`}
-              />
-            </Tabs>
-          }
-          actionButton={
-            <NERButton
-              variant="contained"
-              disabled={isGuest(user.role)}
-              startIcon={<Add />}
-              onClick={() => history.push(routes.CHANGE_REQUESTS_NEW)}
-            >
-              New Change Request
-            </NERButton>
-          }
-        />
-      </Box>
+    <PageLayout
```

Comment:
<img width="726" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/e3d52238-0da0-458c-a45d-658bffd8aa1b">
this happens when you shrink the screen, I think ideally the button would stay in the top right, might be a little weird to get that working tho